service/src/main/java/bio/terra/javatemplate/dao/ExampleDao.java

```diff
@@ -21,7 +21,7 @@ public ExampleDao(NamedParameterJdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
   }
 
-  @Traced
+  @WithSpan // adds this db to the trace
```

Comment:
I don't think a comment is needed here, and I wouldn't want people using the template project to assume that they should be commenting this annotation if they start from this project. 
```suggestion
  @WithSpan
```

Also, does this "add this db to the trace"? I thought it would create a new span that contains this API call. Does opentelemetry recognize `@Repository` classes and do something different for them?